global.json

```diff
@@ -1,18 +1,18 @@
 {
   "sdk": {
-    "version": "6.0.100-preview.3.21202.5",
+    "version": "6.0.100-preview.4.21255.9",
```

Comment:
@dotnet/roslyn-infrastructure This dependency update is failing. Perhaps related to bumping the dotnet sdk version.

Some errors include
- [`The result "" of evaluating the value "$(ArcadeSdkBuildTasksAssembly)" of the "AssemblyFile" attribute in element <UsingTask> is not valid.`](https://dev.azure.com/dnceng/public/_build/results?buildId=1184580&view=logs&j=b2bb0dc1-9cbd-51f0-af1d-56de48e14afe&t=cb3684e9-f47f-55e0-c59e-3841eb9cbe23) (in Build_Windows_Debug, Build_Windows_Release, and Correctness_Rebuild)
- [`Version 6.0.100-preview.4.21255.9 of the .NET SDK requires at least version 16.10.0 of MSBuild.`](https://dev.azure.com/dnceng/public/_build/results?buildId=1184580&view=logs&j=5e4c1a10-89a7-5bf1-0765-c10b6cf6bdb8&t=8490a8da-5b3c-5ac4-77a4-358fbd282c33) (in Correctness_Determinism and Correctness_Build)

Reply:
If we have to update the "vs" specified in our global.json to 16.10, this will cause us issues in our official build. We currently take advantage of a bug in the 16.8 MSBuild to not get failing builds with the MVID mismatch warnings and errors.

@genlu Have you been able to make any progress on diagnosing the cause of those errors?

Reply:
I think the latest commit contains this change https://github.com/dotnet/arcade/pull/7599, we should be able to test the new switch now. Just need to set `IgnoreIbcMergeErrors` property to true is Roslyn

Reply:
Opened  https://github.com/dotnet/roslyn/pull/54997/ to make this change and update Arcade